src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form data-qa="big" action="https://www.google.com/">
+      <label for="icon_key_big" class="search-bar-big" >
+        <img
+          class="icon_big" 
+          src="images/Search.svg" 
+          alt="icon_big"
+        >
+        <input 
+          id="icon_key_big"
+          data-qa="keypress"
+          class="input input_big"
+          type="text" 
+          placeholder="Try &quot;Los Angeles&quot;"
+        > 
+      </label>
+    </form>
+    <form data-qa="small" action="https://www.google.com/">  
+      <label for="icon_key_small" class="search-bar-small">
+        <img
+          class="icon_small" 
+          src="images/Search.svg" 
+          alt="icon_small"
+        >
+        <input 
+          id="icon_key_small" 
+          data-qa="keypress"
+          class="input input_small" 
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+        > 
+      </label>
+    </form>  
```

Comment:
```suggestion
    <form data-qa="big" action="https://www.google.com/">
      <label for="icon_key_big" class="search-bar" >
        <img
          class="icon" 
          src="images/Search.svg" 
          alt="icon_big"
        >
        <input 
          id="icon_key_big"
          data-qa="keypress"
          class="input"
          type="text" 
          placeholder="Try &quot;Los Angeles&quot;"
        > 
      </label>
    </form>
    <form data-qa="small" action="https://www.google.com/">  
      <label for="icon_key_small" class="search-bar search-bar-small">
        <img
          class="icon icon_small" 
          src="images/Search.svg" 
          alt="icon_small"
        >
        <input 
          id="icon_key_small" 
          data-qa="keypress"
          class="input input_small" 
          type="text"
          placeholder="Try &quot;Los Angeles&quot;"
        > 
      </label>
    </form>  
```
lets make it work like this